packages/cbioportal-utils/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "cbioportal-utils",
   "description": "cBioPortal Utilities",
-  "version": "0.1.0",
+  "version": "0.1.1",
```

Comment:
@onursumer should I update version? I need this feature in transcript dropdown pr

Reply:
@leexgh Please revert this change. Package versioning is handled automatically. Once this PR is merged, `publish bot` will bump this version to `0.1.1`, take care of the dependencies, and publish new version(s) to npm.

Reply:
Well... publish bot is broken due to the latest yarn version update. We need to merge cBioPortal/cbioportal-publisher/pull/2 before merging this PR

Reply:
@onursumer thanks for the explanation